v0/destinations/am/transform.js

```diff
@@ -585,7 +586,19 @@ function processSingleMessage(message, destination) {
       break;
     case EventType.TRACK:
       evType = message.event;
-
+      if (!isDefinedAndNotNullAndNotEmpty(evType)) {
```

Comment:
Do we need this taste for a string-type variable?

Reply:
Yes there can be empty strings aswell